src/core/ts/shared-resources/models/ExtensionState.ts

```diff
@@ -342,6 +342,11 @@ export class ExtensionState extends Model({
             path: ["translationRequested"],
             value: true,
           },
+          {
+            op: "replace",
```

Comment:
is there an easy way to cover it in tests?

Reply:
Adding an e2e test for particular steps to reproduce is probably the most idiomatic way to cover these kinds of issues, but there is also ways to test state changes at a low level.